capstone/src/main/resources/static/script.js

```diff
@@ -185,32 +219,68 @@ function createOrgProfile(data) {
   document.getElementById("lastname").style.display = "none";
 }
 
-// function to toggle between the two different forms
-function displayForm(userType) {
+// function used to toggle after a change in the selected user type input
+function toggleForm(formUserType) {
+  var userType = document.getElementById(formUserType).value;
+  displayForm(userType, true);
+}
+
+// helper function for displaying forms
+// a new user will be able to toggle, but a returning user will not
+function displayForm(userType, displayBoth) {
   if (userType == "individual") {
     document.getElementById("user").style.display = "block";
```

Comment:
There is likely an opportunity here to have a helper method `updateUserType(isOrganization)`. Whenever you find code blocks that look almost identical except for some values being set, consider a helper method that takes a parameter differentiating those values.